ex6_3_4.py

```diff
@@ -0,0 +1,12 @@
+def full_names(first_names, last_names, min_length=0)->list:
```

Comment:
Use type hints 
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/